pipeline_steps/common.groovy

```diff
@@ -188,7 +266,11 @@ def conditionalStage(Map args){
   stage(args.stage_name){
     if (env.STAGES.contains(args.stage_name)){
         print "Stage Start: ${args.stage_name}"
-        args.stage()
+        ansiColor('xterm') {
+          timestamps(){
+            args.stage()
+          } //timestamps
+        } //ansiColor
         print "Stage Complete: ${args.stage_name}"
```

Comment:
I thought it made sense to just time the stage thats being executed. If we want timestamps over everything, the call should be at the top level of every job (as timeout is currently).